docs/reference/release-notes/highlights-6.5.0.asciidoc

```diff
@@ -18,3 +18,20 @@ ingest the file into {es}. If you want to influence the output, you can specify
 optional parameters. For example, if the file contains semi-structured text, you 
 can specify a Grok pattern or let the tool generate it. This API is also used by 
 the new File Data Visualizer in {kib}.  
+
+
+[float]
+=== Find multi-bucket anomalies in {ml} jobs
+
+Sometimes events are not interesting or anomalous in the context of a single 
+bucket. However, they become interesting when you take into consideration the 
```

Comment:
maybe just "...consideration a sequence of buckets as a whole"

Reply:
Done, thanks!